addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -287,6 +289,7 @@ test("many2one in form view", async () => {
     expect(".o_avatar_card").toHaveCount(1);
     expect(".o_avatar_card_buttons button:eq(0)").toHaveText("Send message");
     await contains(".o_avatar_card_buttons button:eq(0)").click();
+    await waitFor(".o-mail-ChatWindow");
     await waitFor(".o-mail-ChatWindow-header:contains('Luigi')");
     expect(".o-mail-ChatWindow").toHaveCount(2);
```

Comment:
I don't think this extra assertion maters: the `.o-mail-ChatWindow-header:contains('Luigi')` already asynchronously await for the new chat window to open. The other chat window stays on UI, therefore awaiting asynchronously the new chat window definitely work to assert synchronously 2 chat windows.